com.unity.multiplayer.mlapi/Runtime/NetworkVariable/Collections/NetworkSet.cs

```diff
@@ -104,7 +105,7 @@ public bool IsDirty()
                 return false;
             }
 
-            if (m_NetworkBehaviour.NetworkManager.NetworkTime - LastSyncedTime >= (1f / Settings.SendTickrate))
+            if ((m_NetworkBehaviour.NetworkManager.LocalTime.FixedTime - LastSyncedTime.FixedTime) >= (1f / Settings.SendTickrate))
```

Comment:
Ditto for comment in NetworkList.  I wonder how we can unify Set/List/etc. to not have this same code?